soccer/gameplay/tactics/defense.py

```diff
@@ -71,6 +113,21 @@ def execute_running(self):
             # raise RuntimeError("Defense tactic requires a goalie id to be set")
 
             # TODO: move a lot of this code into modules in the evaluation folder
+
+            #main.system_state().draw_circle(robocup.Point(0, 0), constants.Field.ArcRadius * 2,constants.Colors.Red, "Clear Ball")
+
+    def execute_clearing(self):
+        defender1 = self.subbehavior_with_name('defender1')
+        defender2 = self.subbehavior_with_name('defender2')
+        defender1.go_clear = True
+        defender2.go_clear = False
+
+    def on_exit_clearing(self):
+        defender1 = self.subbehavior_with_name('defender1')
+        defender2 = self.subbehavior_with_name('defender2')
+        defender1.go_clear = False
+        defender2.go_clear = False
```

Comment:
If we're always using the `defender1` behavior as the ball-clearer (which I think is a good idea), we shouldn't have to set `go_clear` to false for `defender2` - it'll always already be false.